tests/test_transpiler_router.py

```diff
@@ -203,15 +202,22 @@ def test_sabre_looping():
 
     placer = Trivial(connectivity=chip)
     initial_layout = placer(loop_circ)
-    router = Sabre(connectivity=chip)
-    routed_circuit, final_layout = router(loop_circ, initial_layout=initial_layout)
+    router_old = Sabre(connectivity=chip, swap_threshold=np.inf)  # Old
+    router_new = Sabre(connectivity=chip)  # New
```

Comment:
Thanks, that's exactly what I meant, I would only change the names because `router_old` and `router_new` are meaningful names for us that we have followed the discussion, but in the future some other people might get confused. I would suggest something like, `router_no_threshold` (for the old one) and `router` for the new one, or something similar.